lms/templates/admin/course/show.html.jinja2

```diff
@@ -0,0 +1,24 @@
+{% import "lms:templates/admin/macros.html.jinja2" as macros %}
+{% extends "lms:templates/admin/base.html.jinja2" %}
+{% block header %}Course {{ course.id }}{% endblock %}
+{% block content %}
+<div>
+    <fieldset>
+        <legend class="label has-text-centered">Course</legend>
+            {{ macros.disabled_text_field("ID", course.id) }}
+            {{ macros.disabled_text_field("Name", course.name) }}
+            {{ macros.created_updated_fields(course) }}
+    </fieldset>
+    
+    <fieldset class="box">
```

Comment:
I don't think we need to search for assignments. In all LMS the assignments are all listed in the course page. We can do the same.